src/app/mainview/mainview.component.ts

```diff
@@ -1405,6 +1412,16 @@ export class MainviewComponent implements OnInit {
         this.unusedRegRegions = this.regressionRegions.filter(entry1 => !this.regRegionsScenarios.some(entry2 => entry1.id === entry2.id));
     }
     
+    public getStatusDescription(sID) {
+        this.tempStatusID = sID;    
+        this.status.forEach(z => {
+            if (this.tempStatusID === z.id) {
+                this.tempStatusName = z.name;
+            }
+        });
+        return (this.tempStatusName);
```

Comment:
If you're not using these variables elsewhere, you don't have to create global variables. You can use either a local variable or use a one line `find()` function and return the response from that. I'll post my guess at both for learning purposes (not testing it out so you may have to adapt).

e.g. with local variable :
```
let statusName;
this.status.forEach(z => {
            if (sID === z.id) {
                statusName = z.name;
            }
        });
return statusName;
```

or 1 line:
```
return this.Status.find(s => s.id == sID).Name;
```